lib/tasks/hyacinth.rake

```diff
@@ -3,15 +3,20 @@
 namespace :hyacinth do
   task reindex: :environment do
     DigitalObjectRecord.find_in_batches(batch_size: (ENV['BATCH_SIZE'] || 1000).to_i) do |records|
-      records.each { |record| ::DigitalObject::Base.find(record.uid).index(false) }
+      records.each do |record|
+        ::DigitalObject::Base.find(record.uid).index(false)
+      rescue => e
+        puts "Error while reindexing #{record.uid}. See raised error below:"
+        raise e
```

Comment:
IIRC you can actually just use an unmapped `rescue` and `raise` - the no arg raise I think raises the last exception

Reply:
I didn't realize you could do that!  I'll make that change.  Thanks!